teckers/teckers/AppDelegate.swift

```diff
@@ -29,7 +29,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     @available(iOS 9.0, *)
```

Comment:
If you are allowing iOS 9.0 we need to implement some test cases, please figuring out an option for this